src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+const listMain = document.querySelector('ul');
+
+function convNumber(text) {
+  const transformTextPhase1 = text.split('');
+
+  transformTextPhase1.shift();
+
+  const transformTextPhase2 = transformTextPhase1.join('');
+  const transformTextPhase3 = transformTextPhase2.split(',').join('');
+  const result = Number(transformTextPhase3);
+
+  return result;
+}
```

Comment:
it can be simplified
```suggestion
function convNumber(text) {
  return Number(text.replace(/[^0-9.-]+/g, ''));
}
```